src/app/components/milestone-table/milestone-table.component.html

```diff
@@ -0,0 +1,77 @@
+<table mat-table class="mat-elevation-z8 demo-table table-resource-laptop">
+  <tr>
+    <th></th>
+    <th>Description</th>
+    <th>Start Date</th>
+    <th>Delivery Date</th>
+    <th>Acceptance Date</th>
+  </tr>
+  <tr *ngFor="let milestone of milestones; let i = index" class="data-row">
+    <td class="data-cell">{{ i + 1 }}</td>
+    <td class="data-cell">{{ milestone.description }}</td>
+    <td class="data-cell">
+      {{
+        this.resourceService.prettyPrintDate(
+          milestone.start_date.toLocaleString("en-GB", { timeZone: "UTC" })
```

Comment:
This all calculation should be in ts file also pre calculated because this will trigger change detection and hamper performance. also if you see this is all repetitive code as you are doing this in multiple place so best to write a method to do this calculation